modules/exploits/unix/webapp/drupal_drupalgeddon2.rb

```diff
@@ -381,7 +326,7 @@ def exploit_drupal7(func, code)
 
     res = send_request_cgi(
       'method'    => 'POST',
-      'uri'       => target_uri.path,
+      'uri'       => normalize_uri(target_uri.path),
```

Comment:
Technically `TARGETURI` is already normalized in `setup`, but this is the only way to be sure.

Reply:
:boom: :earth_americas: